cmd/skaffold/app/cmd/init.go

```diff
@@ -65,6 +66,7 @@ func NewCmdInit() *cobra.Command {
 			{Value: &enableNewInitFormat, Name: "XXenableNewInitFormat", DefValue: false, Usage: "", Hidden: true, IsEnum: true},
 			{Value: &enableJibInit, Name: "XXenableJibInit", DefValue: true, Usage: "", Hidden: true, IsEnum: true},
 			{Value: &enableJibGradleInit, Name: "XXenableJibGradleInit", DefValue: false, Usage: "", Hidden: true, IsEnum: true},
+			{Value: &enableKoInit, Name: "XXenableKoInit", DefValue: true, Usage: "", Hidden: true, IsEnum: true},
```

Comment:
Let's keep the default value false to start with. For most alpha features, we start with flag value false and then  switch it to true.

Reply:
I pushed another commit that sets the flag value to false by default.